crates/winscard/Cargo.toml

```diff
@@ -30,6 +30,8 @@ flate2 = { version = "1.0.28", features = ["zlib", "rust_backend"], default-feat
 rsa = { version = "0.9.6", features = ["hazmat", "sha1"], default-features = false }
 rand_core = "0.6.4"
 sha1 = { version = "0.10.6", default-features = false }
+base64 = { version = "0.21" , optional = true }
+picky-asn1-der = { version = "0.4", optional = true }
```

Comment:
@CBenoit I noticed we have some common dependencies among the `sspi`, `ffi`, and `winscard` crates. Should we move them to workspace dependencies?

Reply:
Yes, why not?  But it’s not very critical, since we use dependabot to update them automatically!